theano/gpuarray/opt.py

```diff
@@ -55,14 +59,34 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
 gpu_cut_copies = EquilibriumDB()
 
+old_not_transferred = []
+new_not_transferred = []
```

Comment:
Can you add a couple of comments explaining what those variables are and why they need to be global?